### ISSUE TITLE
Add binning option in helper functions

### ISSUE THREAD

**Issue by andy-esch** [CartoDB/cartoframes#926]
We should offer the option of setting the number of bins from the helper functions that give histograms (e.g., any that animate). Otherwise, we will get artifacts of mis-binning like below, where each day has very close to the same number of events (that is, it should be roughly flat instead of spiked):
<img width="282" alt="Screen Shot 2019-08-20 at 4 31 14 PM" src="https://user-images.githubusercontent.com/1041056/63382158-210c9a80-c368-11e9-84d5-9cb17bc33084.png">

**Comment by elenatorro** [comment]
Yes! You're totally right. This is a quick change, actually, and I think we must include it.
